integration-tests/src/main/resources/application.properties

```diff
@@ -13,6 +13,7 @@ quarkus.oidc.web-app.authentication.user-info-required=true
 quarkus.oidc.web-app.authentication.cookie-path=/web-app
 quarkus.oidc.web-app.logout.path=/web-app/logout
 quarkus.oidc.web-app.tenant-paths=/web-app,/web-app/logout
+quarkus.oidc.web-app.authentication.allow-multiple-code-flows=false
```

Comment:
This was done to avoid `quarkus-oidc` generating unique state cookie values, to ease the testing